apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -393,376 +344,335 @@ export const PoolAdvertisementPoster = ({
-                              {skill.description
-                                ? skill.description[locale]
-                                : ""}
-                            </Text>
-                          </div>
-                        </Accordion.Content>
-                      </Accordion.Item>
-                    ))}
-                  </Accordion.Root>
-                </>
-              ) : null}
-              {essentialSkills[SkillCategory.Behavioural]?.length ? (
-                <>
-                  <IconTitle icon={CloudIcon}>
-                    {intl.formatMessage({
-                      defaultMessage: "Transferrable skills",
-                      id: "0I8W8B",
-                      description:
-                        "Title for transferrable skills on a pool advertisement",
-                    })}
-                  </IconTitle>
-                  <Text>
-                    {intl.formatMessage({
-                      defaultMessage:
-                        "To be admitted into this process, you will need to display capability in these skills during the assessment process.",
-                      id: "0FjYi+",
-                      description:
-                        "Explanation of a pools required transferrable skills",
-                    })}
-                  </Text>
-                  <Accordion.Root type="multiple">
-                    {essentialSkills[SkillCategory.Behavioural]?.map(
-                      (skill) => (
-                        <Accordion.Item value={skill.id} key={skill.id}>
-                          <StandardAccordionHeader>
-                            {skill.name[locale] || ""}
-                          </StandardAccordionHeader>
-                          <Accordion.Content>
-                            <div data-h2-margin-top="base(x1)">
-                              <Text>
-                                {skill.description
-                                  ? skill.description[locale]
-                                  : ""}
-                              </Text>
-                            </div>
-                          </Accordion.Content>
-                        </Accordion.Item>
-                      ),
-                    )}
-                  </Accordion.Root>
-                </>
-              ) : null}
+              <div data-h2-margin-bottom="base(x3)">
+                <DataRow
+                  Icon={CurrencyDollarIcon}
+                  label={intl.formatMessage({
+                    defaultMessage: "Salary range:",
+                    id: "ls7b2p",
+                    description: "Label for pool advertisement salary range",
+                  })}
+                  value={
+                    localizeSalaryRange(
+                      classification?.minSalary,
+                      classification?.maxSalary,
+                      locale,
+                    ) || notAvailable
+                  }
+                />
```

Comment:
seem to be missing this bit from Figma

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/1ed0d1da-29c5-4de0-b670-bf1490a4dbea)

Reply:
The link displays the same information that is available in the accordion so it was removed as redundant.

![classification](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/d737c12c-d502-4f49-bbf7-05b1331aa66f)